### PR TITLE
Implemented Apprise

### PR DESCRIPTION
**Pull request by ToHindahl**
#50

### CODE REVIEW

File: src/config_generator.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 from common import GOOD_PLZ
 
-NOTIFIERS = ["EMail", "Telegram", "Webbrowser"]
+NOTIFIERS = ["EMail", "Telegram", "APPRISE", "Webbrowser"]
```
Comment (sibalzer) [inline src/config_generator.py:10]: -> Apprise

File: src/settings.py [hunk 2]
```diff
@@ -82,6 +82,24 @@ def load(path: str):
...
+        SEND_APPRISE = True if config["APPRISE"]["enable"].lower() == "true" else False
+
+    except KeyError:
+        log.warning(
+            "[APPRISE] 'enable' is missing in Config. Set False")
+        SEND_APPRISE = False
+
+    try:
+        if SEND_APPRISE:
+            APPRISEOBJ = apprise.Apprise()
```
Comment (sibalzer) [inline src/settings.py:95]: The settings.py should only contain constant default types. Can you move the creation of the apprise object to the send_apprise function in alerts.py?

File: src/config_generator.py (hunk 1 above)
Comment (sibalzer) [inline src/config_generator.py]: APPRISE -> Apprise

File: src/alerts.py [hunk 3]
```diff
@@ -81,3 +92,9 @@ def send_telegram_msg(msg: str) -> None:
             text=f"*{msg}*\n{appointment_url}",
             parse_mode=ParseMode.MARKDOWN
         )
+
+def send_apprise(msg: str) -> None:
+    settings.APPRISEOBJ.notify(
+        body=f"*{msg}*\n{appointment_url}",
+        title='Impftermin Benachrichtigung!',
```
Comment (sibalzer) [inline src/alerts.py:99]: minor: msg as title

**Review by sibalzer — CHANGES_REQUESTED** [review]

**Comment by ToHindahl** [comment]
Okay pushed all changes. I Think

**Review by sibalzer — APPROVED** [review]

**Comment by sibalzer** [comment]
You closed the issue before i could merge it 😕

**Comment by ToHindahl** [comment]
Oh that was not intended. But I cannot reopen this pull request

**Comment by sibalzer** [comment]
You have to fork push & pr again

**Pull request closed without merging**